map/search_mark.cpp

```diff
@@ -182,6 +184,9 @@ class SearchMarkTypeChecker
       {{"tourism", "museum"},        SearchMarkType::Museum},
       {{"amenity", "arts_centre"},   SearchMarkType::Art},
       {{"tourism", "gallery"},       SearchMarkType::Art},
+      {{"tourism", "information"},                      SearchMarkType::Information},
+      {{"tourism", "information", "office"},            SearchMarkType::Information},
+      {{"tourism", "information", "visitor_centre"},    SearchMarkType::Information},
```

Comment:
ToDo: Check here and around if it is really necessary to add parent type and its children.